core/profiling/profiling.h

```diff
@@ -56,6 +56,10 @@
 	new (&__godot_tracy_zone_##m_group_name) tracy::ScopedZone(&TracyConcat(__tracy_source_location, TracyLine), TRACY_CALLSTACK, true)
 #endif
 
+// Memory allocation
+#define GodotProfileAlloc(ptr, size) TracyAlloc(ptr, size)
+#define GodotProfileFree(ptr) TracyFree(ptr)
```

Comment:
```suggestion
#define GodotProfileAlloc(m_ptr, m_size) TracyAlloc(m_ptr, m_size)
#define GodotProfileFree(m_ptr) TracyFree(m_ptr)
```
Per the macro naming standard